open_spiel/games/german_whist_foregame/german_whist_foregame.cc

```diff
@@ -0,0 +1,697 @@
+
+#include "open_spiel/games/german_whist_foregame/german_whist_foregame.h"
+
+#include <filesystem>
```

Comment:
Similarly this is not an approved header. Could you use the utilities library [here](https://github.com/google-deepmind/open_spiel/blob/master/open_spiel/utils/file.h)? I assume that should have everything you need.

Reply:
I think it only uses approved headers now.